### DISCUSSION TITLE
updating old templates

### DISCUSSION THREAD

**Discussion by the-V01D** [pufferpanel/pufferpanel#1321] [General]
if a company updates there services that breaks PufferPanels templates to an extent will the template be updated?

**Comment by the-V01D** [comment]
im asking in regards to Cfx.Re's game FiveM
im not sure if the issue is still there but they added a autocomplete system to their terminal that breaks puffer panel.
will it ever be updated to fix the issue, or are all templates pretty much staying the same as they are?

**Reply by LordRalex** [reply]
That would be an issue specific to that software. 

Templates do get updates when needed. If they have a way to disable autocomplete, that is what we would need to know.

**Reply by the-V01D** [reply]
as far as im aware there is no way do disable it, i first brought it up in the Cfx.Re discord when i couldnt get it to work, they told me to try to run the server directly from the files and see if the panel was the issue. it was.

**Reply by the-V01D** [reply]
i was talking to them today, i think the only way to get pufferpanel to work with this is to make it work with their terminal "system" where it has the autocomplete. they would rather people use their "txAdmin" so they arnt helping people with other panels.

**Reply by LordRalex** [reply]
All we do is hook into stdin and stdout, so there isn't much we can do if they use some odd custom thing.
